src/main/java/org/apache/maven/plugins/javadoc/AbstractFixJavadocMojo.java

```diff
@@ -1412,11 +1413,11 @@ private void addDefaultFieldComment( final StringWriter stringWriter, final Java
                 // CHECKSTYLE_OFF: MagicNumber
                 if ( value.length() < 40 )

```

Comment:
should this check the escaped length instead now, which could be longer?

Reply:
escaped length should be >= original length (if my html knowledges are correct).
I'm OK to change it to escaped length.